changelog.txt

```diff
@@ -1,5 +1,5 @@
+* Fixes bug where profiler table rendering crashes due to undefined `path` fields.
 * Auto-download Firestore and RTDB emulators when using `emulators:start`.
 * Make `functions:shell` respect the `--port` argument.
 * Improve error message when `firebase serve` can't acquire the right port.
-* Allow running the Firestore and RTDB emulators without a configuration.
-* Allow the Firestore emulator to give more information about invalid rulesets.
```

Comment:
please fix

Reply:
I believe I finally got this right in the most recent commit.